encoders/nlp/UniversalSentenceEncoder/__init__.py

```diff
@@ -17,21 +19,29 @@ class UniversalSentenceEncoder(BaseTFEncoder):
     def __init__(
             self,
             model_url: str = 'https://tfhub.dev/google/universal-sentence-encoder/4',
-            *args,
+            preprocessor_url: Optional[str] = None,
+            * args,
             **kwargs):
         """
         :param model_url: the url of the model (TensorFlow Hub). For supported models see
                           family overview: https://tfhub.dev/google/collections/universal-sentence-encoder/1)
+        :param preprocessor_url: the url of preprocessors (TensorFlow Hub).
         :param args:
         :param kwargs:
         """
         super().__init__(*args, **kwargs)
         self.model_url = model_url
+        self.preprocessor_url = preprocessor_url
 
     def post_init(self):
         self.to_device()
+        import tensorflow as tf
+        import tensorflow_text as text
```

Comment:
Unused import?
```suggestion
```

Reply:
It fails if it is not added...
` FileNotFoundError: Op type not registered 'CaseFoldUTF8' in binary running on `